barretenberg/cpp/src/barretenberg/dsl/acir_format/ec_operations.cpp

```diff
@@ -35,10 +35,10 @@ void create_ec_add_constraint(Builder& builder, const EcAdd& input, bool has_val
     cycle_group_ct input2_point(x2, y2, infinite2);
     // Addition
     cycle_group_ct result = input1_point + input2_point;
-
-    auto x_normalized = result.x.normalize();
-    auto y_normalized = result.y.normalize();
-    auto infinite = result.is_point_at_infinity().normalize();
+    cycle_group_ct standard_result = result.get_standard_form();
```

Comment:
Where are some of these changes coming from that are not on master? Did we mean to commit them as part of this PR?

Reply:
You inked from Guis other noir sync one